### PR TITLE
Download imaging data by list of subject id's

### PR DESCRIPTION
**Pull request by mohanadarafe**

### CODE REVIEW

**Comment by glatard** [comment]
Hello @mohanadarafe, thanks for doing this, it seems to be working perfectly, just one thing: could you mutualize code as much as possible between functions `download_metadata` and `download_imaging_data`? For instance the webdriver creation, login and conditional unzipping should be deferred in functions called by `download_metadata` and `download_imaging_data`. Also, `PPMIMetaDataDownloader` could be renamed to `PPMIDownloader`.

File: ppmi_metadata/ppmi_metadata.py [hunk 1]
```diff
@@ -36,6 +36,101 @@ def __init__(self, email, password):
...
+        WebDriverWait(self.driver, timeout).until(download_complete)
+
+        # Move file to cwd or extract zip file
+        downloaded_files = os.listdir(tempdir)
+
+        # we got imaging data and metadata
+        assert(len(downloaded_files) == 2)
+
+        for file_name in downloaded_files:
+            assert(file_name.endswith('.zip') or file_name.endswith('.csv'))
```
Comment (glatard) [inline ppmi_metadata/ppmi_metadata.py:120]: This assertion fails because files are .dcm

Comment (mohanadarafe) [reply ppmi_metadata/ppmi_metadata.py:120]: Good catch

**Comment by mohanadarafe** [comment]
> Hello @mohanadarafe, thanks for doing this, it seems to be working perfectly, just a few things:
> 
> 1. could you mutualize code as much as possible between functions `download_metadata` and `download_imaging_data`? For instance the webdriver creation, login and conditional unzipping should be deferred in functions called by `download_metadata` and `download_imaging_data`.
> 2. `PPMIMetaDataDownloader` could be renamed to `PPMIDownloader`.
> 3. I'm not sure if all the `time.sleep(2)` are needed since `click_button` and `enter_data` will both retry every second.

1. Done
2. Done
3. Some of them are required since Selenium acts faster than PPMI's HTML interface. Sometimes, you need to click on a checkbox which enables another button. However, it happens randomly that the checkbox is clicked very fast before loading the disabled button which leads the bot to being stuck.

Note: I could not refactor the `download_complete` function

@glatard

**Pull request merged**